test/bdd/steps/lib/http-api-helper.js

```diff
@@ -462,6 +462,39 @@ async function apiNodeInfo(nodeRpcUrl) {
     });
 }
 
+/**
+ * @typedef {Object} ApiBalanceInfo
+ * @property {Object} profile info about profile balance
+ * @property {Object} wallet info about wallet balance
```

Comment:
You can specify object's property as well. See http://usejsdoc.org/tags-property.html